genesyscloud/architect_flow/resource_genesyscloud_flow.go

```diff
@@ -113,10 +113,31 @@ func updateFlow(ctx context.Context, d *schema.ResourceData, meta interface{}) d
 	}
 
 	s3Uploader := files.NewS3Uploader(reader, nil, substitutions, headers, "PUT", presignedUrl)
-	_, err = s3Uploader.Upload()
+	fileInfo, err := os.Stat(filePath)
 	if err != nil {
+		log.Printf("failed to read file information. Path: '%s' Error: %v", filePath, err)
+	}
+
+	uploadErr := util.WithRetries(ctx, 20*time.Second, func() *retry.RetryError {
+		uploadStartTime := time.Now()
+		_, err = s3Uploader.Upload()
+		uploadDuration := time.Since(uploadStartTime)
+		if err == nil {
+			return nil
+		}
+		// Retrying when encountering the error from DEVTOOLING-425
+		if strings.Contains(fmt.Sprintf("%v", err), "An existing connection was forcibly closed by the remote host") {
```

Comment:
I am just wondering if we should retry on any error?  It seems like if we have any kind of problem we should retry

Reply:
I'm going to change it because there is also a chance aws could change the error message they return, then this code wouldn't work anymore

Reply:
Done